dev/puppet-server.conf.sample

```diff
@@ -65,6 +65,18 @@ http-client: {
     #                TLS_RSA_WITH_AES_256_CBC_SHA,
     #                TLS_RSA_WITH_AES_128_CBC_SHA256,
     #                TLS_RSA_WITH_AES_128_CBC_SHA]
+
+    # The amount of time, in milliseconds, that an outbound HTTP connection
+    # will wait for data to be available before closing the socket. If not
+    # defined, defaults to 20 minutes. If 0, the timeout is infinite and if
+    # negative, the value is undefined (system default).
```

Comment:
Probably would be good to have the same text here as below for `connect-timeout-milliseconds`:

> if negative, the value is undefined in the application and governed by the system default behavior.